src/rest-client-v5.ts

```diff
@@ -563,6 +574,24 @@ export class RestClientV5 extends BaseRestClient {
     });
   }
 
+  /**
+   * This endpoint allows you to set the disconnection protect time window. Covers: option (unified account).
+   *
+   * If you need to turn it on/off, you can contact your client manager for consultation and application.
+   * The default time window is 10 seconds.
+   *
+   * Only for institutional clients!
+   */
+  setDisconnectCancelAllWindowV2(
+    product: 'OPTION' | 'SPOT' | 'DERIVATIVES',
+    timeWindow: number,
+  ): Promise<APIResponseV3<undefined>> {
+    return this.postPrivate('/v5/order/disconnected-cancel-all', {
+      product,
+      timeWindow,
+    });
+  }
```

Comment:
Please use object params to reduce future pains.
```suggestion
  setDisconnectCancelAllWindowV2(params: { 
    product: 'OPTION' | 'SPOT' | 'DERIVATIVES',
    timeWindow: number,
  }): Promise<APIResponseV3<undefined>> {
    return this.postPrivate('/v5/order/disconnected-cancel-all', params);
  }
```